src/azure-cli/HISTORY.rst

```diff
@@ -45,6 +46,10 @@ Release History
 
 * Support for Policy new API version 2019-01-01
 
+**Storage**
+
+`az storage remove`: remove --auth-mode argument
```

Comment:
Is it a breaking change? Removing an existed argument might be harmful. I prefer offering more information in the History.rst.

Reply:
Right. I am thinking about it, too. I will add [BREAKING CHANGE] label.